shell/platform/darwin/ios/ios_context_metal.mm

```diff
@@ -34,6 +35,19 @@
     return;
   }
 
+  CVMetalTextureCacheRef texture_cache_raw = NULL;
+  auto cv_return = CVMetalTextureCacheCreate(kCFAllocatorDefault,  // allocator
+                                             NULL,           // cache attributes (NULL default)
+                                             device_.get(),  // metal device
+                                             NULL,           // texture attributes (NULL default)
+                                             &texture_cache_raw  // [out] cache
+  );
+  if (cv_return != kCVReturnSuccess) {
+    FML_LOG(ERROR) << "Could not create Metal texture cache.";
```

Comment:
What does this mean for a Flutter developer? What action should be taken if any?

Reply:
Hah.  Docs are woefully lacking here, apparently it just documents that when it succeeds it returns this and nothing else... https://developer.apple.com/documentation/corevideo/1456774-cvmetaltexturecachecreate?language=objc

Reply:
should this really be FML_DLOG?

Reply:
> What action should be taken if any?

If I saw that in the a bug report, I'd know immediately where what went wrong and where. If a user filed an error for this with no logs, I probably wouldn't find the cause unless I reproduced the issue and set various breakpoint (if the issue caught my attention at all).

I am on the fence about its use. Probably means I should just get rid of it. Patching.